src/nft/components/collection/CollectionSearch.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from 'nft/components/Box'
+import { useCollectionFilters } from 'nft/hooks/useCollectionFilters'
+import { FormEvent } from 'react'
+
+export const CollectionSearch = () => {
+  const setSearchByNameText = useCollectionFilters((state) => state.setSearch)
+  const searchByNameText = useCollectionFilters((state) => state.search)
+
+  return (
+    <Box
+      as="input"
+      borderColor={{ default: 'medGray', focus: 'genieBlue' }}
+      borderWidth="1px"
+      borderStyle="solid"
+      borderRadius="12"
+      padding="12"
+      backgroundColor="white"
+      fontSize="16"
+      height="44"
+      color={{ placeholder: 'darkGray', default: 'blackBlue' }}
+      value={searchByNameText}
```

Comment:
useDebounce to wait till after the user stops typing. Can check example with universal searchbar